cmd/kubectl-gadget/utils/trace.go

```diff
@@ -229,7 +229,11 @@ RetryLoop:
 		nodeErrors := make(map[string]string)
 		for _, i := range results.Items {
 			if i.Status.State == "Completed" || i.Status.State == "Started" {
-				successNodeCount++
+				if i.Status.Output == "" && i.Spec.OutputMode == "Status" {
+					contextLogger.Debugf("Ignoring empty output from node %q", i.Spec.Node)
```

Comment:
Is this printed in case of `--json`?

Reply:
It should and I think it shouldn't, even though I didn't find how to show debug logs 😅 . I think I will move that log into a comment, do you agree?

Reply:
I think we didn't add the CLI flags for that .. 
There is an example of that at https://github.com/kinvolk/traceloop/blob/main/traceloop.go#L50
But that's outside the scope of this PR.

Your solution looks fine to me

Reply:
Understood. Yes, that's something we can evaluate if could be useful to be added later on. 

PR updated with the comment.